opendbc/car/body/carstate.py

```diff
@@ -30,10 +31,12 @@ def update(self, cp, *_) -> structs.CarState:
     return ret
 
   @staticmethod
-  def get_can_parser(CP):
+  def get_can_parsers(CP):
     messages = [
       ("MOTORS_DATA", 100),
       ("VAR_VALUES", 10),
       ("BODY_DATA", 1),
     ]
-    return CANParser(DBC[CP.carFingerprint]["pt"], messages, 0)
+    return {
+      'main': CANParser(DBC[CP.carFingerprint]["main"], messages, 0)
+    }
```

Comment:
I don't generally like furthering the string dictionary keys for this stuff, an enum sounds much better.

Reply:
guess we can define an enum with some standard bus names somewhere